### PR TITLE
[MSINTEROP-783] Pipelines/Serverless Product to Product Scenario

### PR DESCRIPTION
**Pull request by calebevans**

### CODE REVIEW

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.14-openshift-pipelines-p2p-rosa-classic-openshift-pipelines-serverless-rosa-classic-rc

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-openshift-pipelines-release-tests-release-v1.15-openshift-pipelines-p2p-rosa-classic-openshift-pipelines-serverless-rosa-classic-rc

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      AWS_REGION: us-east-1
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+              [
+                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:50]: How about these changes? if operator-install fails, then this failures shouldn't get reported I guess, wdyt?
```suggestion
                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "p2p-tests", "priority": 2}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
```

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    - ref: openshift-pipelines-deploy-tekton
+    - ref: openshift-pipelines-tests
+    - ref: p2p-additional-labels
+    workflow: firewatch-rosa-aws-sts
+- as: openshift-pipelines-serverless-rosa-classic-rc
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      CHANNEL_GROUP: candidate
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:78]: shouldn;t channel group for 4.17 be `nightly` here?

**Review by chetna14manku — COMMENTED** [review]
Added some comments, otherwise looks good!

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml (hunk 2 above)
Comment (calebevans) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:78]: I think we were just going to do nightly for development, then switch to candidate before merging. @rujutashinde can you confirm?

Comment (rujutashinde) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:78]: yes that is correct nightly for development work and we switch to candidate when we are ready to run

Comment (chetna14manku) [reply ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:78]: okay, got it, thanks for clarifying

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      FIREWATCH_CONFIG: |
+        {
+          "failure_rules":
+              [
+                {"step": "openshift-pipelines-tests", "failure_type": "all", "classification": "Product to Product Test Failure", "jira_project": "SRVKP", "group": {"name": "lp-tests", "priority": 1}, "jira_additional_labels": ["!default"], "jira_epic": "!default"}
+              ],
+          "success_rules": [{"jira_epic": "MSINTEROP-1162"}]
+        }
+      FIREWATCH_CONFIG_FILE_PATH: https://raw.githubusercontent.com/CSPI-QE/cspi-utils/main/firewatch-base-configs/rosa/lp-interop.json
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:87]: Can you also update these same as above

File: ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: openshift-pipelines-serverless-rosa-classic-iib
```
Comment (chetna14manku) [inline ci-operator/config/openshift-pipelines/release-tests/openshift-pipelines-release-tests-release-v1.15__openshift-pipelines-p2p-rosa-classic.yaml:40]: hey, are we not adding ocp version in job names?

**Review by VeereshAradhya — APPROVED** [review]

**Pull request merged**